pkg/cloudprovider/azure.go

```diff
@@ -51,6 +55,10 @@ type Azure struct {
 	nodeMapLock                  sync.Mutex
 	nodeLockMap                  map[string]*sync.Mutex
 	azureWorkloadIdentityEnabled bool
+	cloudPrivateIPConfigLister   cloudnetworklisters.CloudPrivateIPConfigLister
+	nodeLister                   corelisters.NodeLister
+	lbBackendPoolSynced          map[string]bool
```

Comment:
What is the point of `lbBackendPoolSynced` and `lbBackendPoolSyncedLock`? Is it a leftover from the previous approach?